Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8273,6 +8273,7 @@ def create_outputs(whois_result, domain, query=None):
     ec = {'Name': domain}
     standard_ec = {}  # type:dict
     standard_ec['WHOIS'] = {}
+    standard_ec['QueryResult'] = 'NOT FOUND' not in str(whois_result.get('raw', 'NOT FOUND'))
```

Comment:
the issue says the output should be `Not Found` or `Found`, not a boolean

Reply:
Spoke to @David-BMS about it and agreed, I'll write it in the issue